packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -50,7 +60,12 @@ PictogramItem.propTypes = {
   heading: PropTypes.string.isRequired,
   copy: PropTypes.string.isRequired,
   cta: PropTypes.object,
-  Pictogram: PropTypes.object.isRequired,
+  pictogram: PropTypes.shape({
+    src: PropTypes.object.isRequired,
+    ariaLabel: PropTypes.string,
+    ariaHidden: PropTypes.string,
```

Comment:
I'm not sure this is necessary. We're not expecting them both there at the same time. We can demonstrate it in the documentation, and storybook

Reply:
I guess the abstraction of `isRequired` is making it just to make sure to have the right type

Reply:
Wouldn't we then need to do that for other attributes then?